pointpats/quadrat_statistics.py

```diff
@@ -384,7 +387,7 @@ class QStatistic:
                         simulation based inference. Default is 0
                         where no simulation based inference is
                         performed.
-                        
+
     Attributes
     ----------
     pp                : :class:`.PointPattern`
```

Comment:
Looking good! Can you also change the docstrings and maybe add a small example for the addition?

Reply:
Sure, docstring is easy. Where would you like the example?